lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -143,34 +132,17 @@ enum InternalJointMatrixLayout {
 
 enum InternalJointMatrixUse { MatrixA = 0, MatrixB = 1, Accumulator = 2 };
 
-enum InternalJointMatrixCTI {
-  None = 0,
-  TF32 = 1,
-  Bfloat16 = 2,
-  PackedInt2 = 3,
-  PackedInt4 = 4
-};
+enum InternalJointMatrixCTI { None = 0, TF32 = 1, Bfloat16 = 2 };
```

Comment:
May have misread the PR, but I can't see that this enum is actually used after legacy code removal. Do we need it?